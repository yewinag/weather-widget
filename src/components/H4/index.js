import styled from 'styled-components';

const H4 = styled.h4`
    font-weight: bold;
    font-size: 24px;
`
export default H4;