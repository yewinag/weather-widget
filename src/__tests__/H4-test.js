import React from 'react';

import renderer from 'react-test-renderer';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

import H4 from '../components/H4';

Enzyme.configure({ adapter: new Adapter() });

describe('header h4', () => {
    it('render h4', () => {
        const wrapper = shallow(<H4 />)
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });
});