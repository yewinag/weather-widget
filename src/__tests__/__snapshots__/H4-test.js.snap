// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header h4 render h4 1`] = `ShallowWrapper {}`;
